AssetInformationApi.Tests/V1/E2ETests/Stories/AddNewAssetTests.cs

```diff
@@ -0,0 +1,56 @@
+using AssetInformationApi.Tests.V1.E2ETests.Fixtures;
+using AssetInformationApi.Tests.V1.E2ETests.Steps;
+using Hackney.Core.Testing.DynamoDb;
+using System;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace AssetInformationApi.Tests.V1.E2ETests.Stories
+{
+    [Story(
+        AsA = "Service",
+        IWant = "an endpoint to add a new asset and return asset details",
+        SoThat = "it is possible to view the details of an asset.")]
+    [Collection("AppTest collection")]
+    public class AddNewAssetTests : IDisposable
+    {
+        private readonly IDynamoDbFixture _dbFixture;
+        private readonly AddNewAssetSteps _steps;
+        private readonly AssetsFixture _assetsFixture;
+
+        public AddNewAssetTests(MockWebApplicationFactory<Startup> appFactory)
+        {
+            _dbFixture = appFactory.DynamoDbFixture;
+            _assetsFixture = new AssetsFixture(_dbFixture);
+            _steps = new AddNewAssetSteps(appFactory.Client, _dbFixture);
+        }
+
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        private bool _disposed;
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposing && !_disposed)
+            {
+                if (null != _assetsFixture)
+                    _assetsFixture.Dispose();
+
+                _disposed = true;
+            }
+        }
+
+        [Fact]
```

Comment:
Please add more tests for Bad Request and NotFound scenarios

Reply:
Added for BadRequest, notfound is not valid for Post end points